src/main/java/com/questhelper/quests/animalmagnetism/AnimalMagnetism.java

```diff
@@ -282,6 +283,16 @@ public QuestPointReward getQuestPointReward()
 		return new QuestPointReward(1);
 	}
 
+	@Override
+	public List<ExperienceReward> getExperienceRewards()
+	{
+		return Arrays.asList(
+				new ExperienceReward(Skill.WOODCUTTING, 2500),
```

Comment:
Woodcutting should be at the bottom for alphabetization.

Reply:
Fixed! I followed the order it shows on the quest completed screen. Will keep alphabetization in mind next time. Thank you :)